sys/parsoid.js

```diff
@@ -816,6 +817,25 @@ module.exports = (options) => {
                     valueType: 'blob',
                     default_time_to_live: options.grace_ttl
                 }
+            },
+            {
```

Comment:
It would be better if the parsoid module exported only one set of resources - those needed by it. By having both js and php exporting both variants we double the number of requests to Cassandra, and that's not good.

Reply:
That's what I had initially but it was throwing exceptions in the tests (due to a bad reference). Fixed.